src/dao/interfaces.ts

```diff
@@ -12,3 +12,21 @@ export interface DynamodbParams {
 export interface DynamodbResponseBatch {
   UnprocessedItems: object
 }
+
+export interface Operators {
```

Comment:
I was wondering.....is it correct to put dynamo utils and SQL utils without differentiating the two?

Reply:
You can put SqlOperators, what do you think?

Reply:
I Agree with that, adjusted